pylintrc

```diff
@@ -59,7 +59,7 @@ disable=invalid-name,protected-access,fixme,too-many-branches,
         import-error,too-many-lines,too-many-instance-attributes,
         too-many-public-methods,duplicate-code,broad-except,
         redefined-builtin,anomalous-backslash-in-string,
-        missing-docstring,no-member,
+        missing-docstring,no-member,mutable-sequence-modified-in-loop,
```

Comment:
I'm fairly sure we'll be reluctant to merge a check that gives a false-positive even for our own code base.